spec/jobs/export_structural_job_spec.rb

```diff
@@ -1,5 +1,6 @@
 # frozen_string_literal: true
 
+require 'csv'
```

Comment:
Maybe this should this go in `rails_helper`? (I'm not entirely clear on this convention.)

Reply:
I'm surprised it's needed at all, it's required in the app code.

Reply:
The test failed without it.

Reply:
@jcoyne you were right, the require doesn't seem needed -- it's weird because I thought I saw it fail before.  I've removed the require.